src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
+<dt markdown="1">- Display 1 Minute</dt>
+<dt markdown="1">- Display 5 Minutes</dt>
+<dt markdown="1">- Display 10 Minutes</dt>
+<dt markdown="1">- Display All Minutes (slider disabled)</</dt>
+</dd>
+<dt markdown="1">**Source**</dt>
+<dd>Source can be either "Live Feed" current connected running Flutter application or offline "Export" file.</dd>
+<dt markdown="1">**Android Memory**</dt>
+<dd>Displays or hides the Android Memory Chart.</dd>
+<dt markdown="1">**GC**</dt>
+<dd>Initiates a garbage collection - compaction of the heap</dd>
+<dt markdown="1">**Export**</dt>
+<dd>Saves collected data for Event Timeline, Memory Overview Chart and Android Overview Chart. Files saved are displayed under the Source drop-down. Selecting a file will load the offline data.</dd>
+</dl>
+
+## Memory Actions
+
+Below the memory charts (Event Timeline, Memory Overview and Android Overview charts) are actions to collect information about memory e.g., Snapshots, Table Tree view, Treemap view, Groupings, or Monitor Allocations, Searching and Filtering.
+
+![Screenshot of a memory actions]({% asset tools/devtools/memory_actions.png @path %}){:width="100%"}
+
+<dl markdown="1">
 <dt markdown="1">**Snapshot**</dt>
-<dd markdown="1">Display a table of current active
-    memory objects (see
-    [Snapshot classes](#snapshot-classes)).
-<dt markdown="1">**Reset accumulator**</dt>
-<dd markdown="1">Reset values, to zero, under the
-    Accumulator column in the Snapshot classes table.
-<dt markdown="1">**Filtering classes**</dt>
-<dd>PENDING</dd>
-<dt markdown="1">**Snapshot classes**</dt>
-<dd>Clicking the Snapshot button (top right area)
-    displays a table of current memory objects.
-    The memory objects can be sorted by class name,
-    size, allocated instances, and so on.</dd>
-<dt markdown="1">**Accumulator counts since reset**</dt>
-<dd>Clicking the Reset button (top right area) resets
-    the accumulated instances count. Clicking Snapshot
-    after a reset displays the number of new instances
-    allocated since last reset.
-    This is useful for finding memory leaks.</dd>
-<dt markdown="1">**Class instances**</dt>
-<dd>Clicking a class in the Snapshot class table
-    displays the number of active instances for that class.</dd>
-<dt markdown="1">**Inspecting contents of an instance**</dt>
-<dd>PENDING</dd>
-<dt markdown="1">**Total active objects and classes in the heap**</dt>
-<dd>Total classes allocated in the heap and total objects
-    (instances) in the heap.</dd>
-
-## Memory overview chart
+<dd markdown="1">Display a table of all current active
+    memory objects.  Clicking the Snapshot button makes a request to the Dart VM to collect the current state of memory.  A snapshot displays a table of current memory objects. The memory objects can be sorted by class name, size, allocated instances, and so on.(see [Snapshot classes](#snapshot-classes)).
+<dt markdown="1">**Treemap**</dt>
```

Comment:
make one section for Table and one for Treemap

Reply:
Done.